docs/src/modules/components/AppContent.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
All the documentation references it this way: https://flow.org/en/docs/react/components/

Specifically addressed in the note here: https://flow.org/en/docs/react/children/